packages/compass-components/src/hooks/use-confirmation.spec.tsx

```diff
@@ -88,6 +88,7 @@ describe('use-confirmation', function () {
         description: 'This action can not be undone.',
         buttonText: 'Yes',
       });
+      await new Promise((resolve) => setTimeout(resolve, 500));
```

Comment:
Missed this when applying more changes to the branch, will remove

Reply:
Done in https://github.com/mongodb-js/compass/pull/5400/commits/c3ec366a08f1fb18accd93ff8dde97b425e7eda8